cypress/e2e/pages/onDemandAudio/index.cy.js

```diff
@@ -1,32 +1,487 @@
-import config from '../../../support/config/services';
-import getPaths from '../../../support/helpers/getPaths';
-import serviceHasPageType from '../../../support/helpers/serviceHasPageType';
+import runTestsForPage from '#nextjs/cypress/support/helpers/runTestsForPage';
 import e2eTests from './tests';
-import visitPage from '../../../support/helpers/visitPage';
-import { overrideRendererOnTest } from '../../../support/helpers/onDemandRadioTv';
 
 const pageType = 'onDemandAudio';
-Object.keys(config)
-  .filter(service => serviceHasPageType(service, pageType))
-  .forEach(serviceId => {
-    // eslint-disable-next-line prefer-const
-    let { variant, name: service } = config[serviceId];
 
-    const paths = getPaths(serviceId, pageType);
-    paths.forEach(currentPath => {
-      describe(`${pageType} - ${currentPath}`, () => {
-        before(() => {
-          Cypress.env('currentPath', currentPath);
+const tests = [e2eTests];
+const testSuites = [
+  {
+    path: '/afaanoromoo/bbc_afaanoromoo_radio/programmes/w13xttnw', // On Demand Brand
+    service: 'afaanoromoo',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/afaanoromoo/bbc_afaanoromoo_radio/w3ct1wc0', // On Demand Episode
+    service: 'afaanoromoo',
+    runforEnv: ['live'],
+    tests,
+  },
+  {
+    path: '/afaanoromoo/bbc_afaanoromoo_radio/w3ct1wd4', // On Demand Episode
+    service: 'afaanoromoo',
+    runforEnv: ['live'],
+    tests,
+  },
+  {
+    path: '/afrique/bbc_afrique_radio/programmes/p030s6dq', // On Demand Brand,
+    service: 'afrique',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/afrique/bbc_afrique_radio/w172y1g3d9108lh', // On Demand Episode
+    service: 'afrique',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/amharic/bbc_amharic_radio/programmes/w13xttnt', // On Demand Brand,
+    service: 'amharic',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/amharic/bbc_amharic_radio/w3ct1lqy', // On Demand Episode
+    service: 'amharic',
+    runforEnv: ['test', 'live'],
+    tests,
+  },
+  {
+    path: '/arabic/bbc_arabic_radio/programmes/p030vh2y', // On Demand Brand
+    service: 'arabic',
+    runforEnv: 'live',
+    tests,
+  },
+  {
+    path: '/arabic/bbc_arabic_radio/programmes/p030vh25', // On Demand Brand
+    service: 'arabic',
+    runforEnv: 'test',
+    tests,
+  },
```

Comment:
Because test points to live data, we can pick one path and combine them into a single entry, with runforEnv: ['test', 'live']